src/ardrive.ts

```diff
@@ -64,6 +78,31 @@ export interface DriveUploadBaseCosts {
 const stubTransactionID = '0000000000000000000000000000000000000000000';
 const stubEntityID = '00000000-0000-0000-0000-000000000000';
 
+export function lastFolderRevisionFilter(
```

Comment:
All of these filtering functions need docblocks explaining their intent, particularly `childrenAndFolderOfFilterFactory`.

Reply:
Added at: 793293a181598ddce21c2fccb879536623413b62